src/test/java/com/littletech/swarmrestapi/OpenAIApiControllertest.java

```diff
@@ -0,0 +1,52 @@
+package com.littletech.swarmrestapi;
+
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import com.ltbc.swarmrestapi.service.SwarmRestApiService;
+
+import java.io.IOException;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@ExtendWith(MockitoExtension.class)
+public class OpenAIApiControllertest {
```

Comment:
OpenAIApiControllertest -> OpenAIApiControllerTest

Reply:
Ah that's probably why it is broken lol I didn't look at the tests at all but this makes sense

Reply:
I ended up fixing unit tests here so this should be resolved now - changed a few things though but this PR should be shippeable now